heterarchy.litcoffee

```diff
@@ -92,13 +103,13 @@ it in the later.
             value
         else if value is oldklass::constructor and inherited(oldklass) is Object
             superctor = inherited(newklass)::constructor
-            ->
+            () ->
```

Comment:
Sorry about the parens ^^ For function and constructors I **personally** dislike not writing them. But this that's your repo I would change 'em back 😉